guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -9,22 +9,31 @@ The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap
+# {foreman-installer} --enable-foreman-plugin-openscap  --enable-foreman-proxy-plugin-openscap
 ----
-. Install the OpenSCAP plug-in on your {SmartProxies}:
+ifndef::satellite[]
+. Optional: Install the OpenSCAP Hammer CLI plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-proxy-plugin-openscap
+# {foreman-installer} --enable-foreman-cli-openscap
 ----
+endif::[]
+
+. Install the OpenSCAP plug-in on any {SmartProxies}:
 +
-Perform this command on both your {ProjectServer} and any attached {SmartProxies}.
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-proxy-plugin-openscap
+----
+ifndef::satellite[]
 . Optional: Install the OpenSCAP Hammer CLI plug-in:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} --enable-foreman-cli-openscap
 ----
+endif::[]
```

Comment:
Please drop this (line 29 - 36). Installing the CLI is invalid on Smart Proxies and it's already documented above for the main server.